internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +566,42 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_instrumentation) {
+    echo "Installing SignalFx Instrumentation for .NET ..."
+    $api = "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    $module_name = "Splunk.SignalFx.DotNet.psm1"
+    echo "Downloading .NET Instrumentation installer ..."
+    $download = (Invoke-WebRequest $api | ConvertFrom-Json).assets | Where-Object { $_.name -like $module_name } | Select-Object -Property browser_download_url,name
+    $dotnet_auto_path = Join-Path $env:temp $download.name
```

Comment:
There might be a chance that the temp path is somehow deleted. I ran into this issue with one of my VMs. 
Here is the fix I did for SignalFx and similar to OpenTelemetry.
https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/Splunk.SignalFx.DotNet.psm1#L7

Reply:
Looks like this scenario is handled on line 161. I've updated this code to use `$tempdir`.